tests/e2e/lib/ContainerFile.template

```diff
@@ -33,6 +33,7 @@ RUN dnf install -y \
 	gpgme-devel \
 	libseccomp-devel \
 	iptables-services \
+	iptables-nft \
```

Comment:
If you replace both
You can skip L68,

Tried it 
```
podman exec -it control bash -c "iptables --version"
iptables v1.8.8 (nf_tables)
```

Reply:
@Yarboa what you meant replace both?

Reply:
```suggestion
	iptables-nft \
	iptables-services \
```

Reply:
@dougsland ^^